### ISSUE TITLE
Metric Evaluation Code

### ISSUE THREAD

**Issue by saifkhichi96** [bednarikjan/texless_defsurf_recon#1]
Hi. I am wondering if you have the code you used to evaluate the metrics which you use to report results for your experiments in the paper? As far as I can see, this repository only contains the training code, not testing.
